ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/ImageCvePage.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { gql, useQuery } from '@apollo/client';
+import { Breadcrumb, BreadcrumbItem, Divider, PageSection, Skeleton } from '@patternfly/react-core';
+import { useParams } from 'react-router-dom';
+
+import BreadcrumbItemLink from 'Components/BreadcrumbItemLink';
+import { getOverviewCvesPath } from './searchUtils';
+import WorkloadTableToolbar from './WorkloadTableToolbar';
+import ImageCvePageHeader, {
+    ImageCveMetadata,
+    imageCveMetadataFragment,
+} from './ImageCvePageHeader';
+import ErrorSection from './ErrorSection';
+
+const workloadCveOverviewImagePath = getOverviewCvesPath({
+    cveStatusTab: 'Observed',
+    entityTab: 'Image',
+});
+
+export const imageCveMetadataQuery = gql`
+    ${imageCveMetadataFragment}
+    query getImageCveMetadata($cve: String!) {
+        imageCVE(cve: $cve) {
+            ...ImageCVEMetadata
+        }
+    }
+`;
+
+function ImageCvePage() {
+    const { cveId } = useParams();
+    const metadataRequest = useQuery<{ imageCVE: ImageCveMetadata }, { cve: string }>(
+        imageCveMetadataQuery,
+        { variables: { cve: cveId } }
+    );
+
+    const cveName = metadataRequest.data?.imageCVE?.cve;
+
+    return (
+        <>
+            <PageSection variant="light" className="pf-u-py-md">
+                <Breadcrumb>
+                    <BreadcrumbItemLink to={workloadCveOverviewImagePath}>CVEs</BreadcrumbItemLink>
+                    {!metadataRequest.error && (
+                        <BreadcrumbItem isActive>
+                            {cveName ?? (
+                                <Skeleton screenreaderText="Loading image name" width="200px" />
+                            )}
+                        </BreadcrumbItem>
+                    )}
+                </Breadcrumb>
+            </PageSection>
+            <Divider component="div" />
+            <PageSection variant="light">
+                {metadataRequest.error ? (
+                    <ErrorSection error={metadataRequest.error}>
+                        The system was unable to load metadata for this CVE
+                    </ErrorSection>
+                ) : (
+                    // Don't check the loading state here, since if the passed `data` is `undefined` we
+                    // will implicitly handle the loading state in the component
+                    <ImageCvePageHeader data={metadataRequest.data?.imageCVE} />
+                )}
+            </PageSection>
+            <Divider component="div" />
+            <PageSection className="pf-u-display-flex pf-u-flex-direction-column pf-u-flex-grow-1">
+                <WorkloadTableToolbar />
```

Comment:
i wonder if it's worth rendering this even when there is an error above -- looking your last screenshot, it feels a little awkward to have an error and then functional elements below it